src/main/scala/io/iohk/ethereum/crypto/ECIESCoder.scala

```diff
@@ -0,0 +1,165 @@
+package io.iohk.ethereum.crypto
+
+import java.io.{ByteArrayInputStream, ByteArrayOutputStream, IOException}
+import java.math.BigInteger
+import java.security.SecureRandom
+
+import org.spongycastle.asn1.sec.SECNamedCurves
+import org.spongycastle.asn1.x9.X9ECParameters
+import org.spongycastle.crypto.digests.{SHA1Digest, SHA256Digest}
+import org.spongycastle.crypto.engines.AESFastEngine
+import org.spongycastle.crypto.generators.{ECKeyPairGenerator, EphemeralKeyPairGenerator}
+import org.spongycastle.crypto.macs.HMac
+import org.spongycastle.crypto.modes.SICBlockCipher
+import org.spongycastle.crypto.params._
+import org.spongycastle.crypto.parsers.ECIESPublicKeyParser
+import org.spongycastle.crypto.{BufferedBlockCipher, InvalidCipherTextException}
+import org.spongycastle.math.ec.ECPoint
+
+object ECIESCoder {
+  val KEY_SIZE = 128
+  val params: X9ECParameters = SECNamedCurves.getByName("secp256k1")
+  val CURVE = new ECDomainParameters(params.getCurve, params.getG, params.getN, params.getH)
```

Comment:
Do you think we could use the naming conventions as proposed here: http://docs.scala-lang.org/style/naming-conventions.html#constants-values-variable-and-methods? That is upper camel case.